todolist/settings.py

```diff
@@ -62,10 +62,15 @@
 # https://docs.djangoproject.com/en/1.7/ref/settings/#databases
 
 DATABASES = {
-    "default": {
-        "ENGINE": "django.db.backends.sqlite3",
-        "NAME": os.path.join(BASE_DIR, "db.sqlite3"),
+    'default': {
+        'ENGINE': 'mysql.connector.django',
+        'NAME': 'app_db',
+        'USER': 'app_user',
+        'PASSWORD': '1234',
+        'HOST': '172.17.0.2',  # You can use a different host in your MySQL server is on a remote machine.
```

Comment:
Using a hardcoded IP address for the database `HOST` may not be suitable for all deployment environments. Consider using environment variables to configure the database connection dynamically.